Create new DB column publications.db_connection (#745)
LayerManager/layman: CHANGELOG.md

```
### Migrations and checks
#### Schema migrations

> - [#703](https://github.com/LayerManager/layman/issues/703) Create new json column `db_connection` in `publication` table.

Reviewer: ```suggestion
- [#703](https://github.com/LayerManager/layman/issues/703) Create new json column `db_connection` in `publications` table in prime DB schema.
```

#### Data migrations

> - [#703](https://github.com/LayerManager/layman/issues/703) Fill column `db_connection` in `publications` table in prime DB schema for all publications. Value is set to `null` for all publications as it is meaningless for maps and raster layers, or they are internal vector layers.

Reviewer: ```suggestion
- [#703](https://github.com/LayerManager/layman/issues/703) Fill column `db_connection` in `publications` table in prime DB schema for all publications. Value is set to `null` for all existing publications.
```
